instrumentation/opentelemetry-instrumentation-mysql/src/opentelemetry/instrumentation/mysql/__init__.py

```diff
@@ -77,22 +77,24 @@ def _uninstrument(self, **kwargs):
         dbapi.unwrap_connect(mysql.connector, "connect")
 
     # pylint:disable=no-self-use
-    def instrument_connection(self, connection):
+    def instrument_connection(self, connection, tracer_provider=None):
         """Enable instrumentation in a MySQL connection.
 
         Args:
             connection: The connection to instrument.
+            tracer_provider: The optional TracerProvider to use. If omitted
+            the current globally configured one is used.
 
         Returns:
             An instrumented connection.
         """
-        tracer = get_tracer(__name__, __version__)
-
         return dbapi.instrument_connection(
-            tracer,
```

Comment:
Was this a bug?

Reply:
Yes, I don't know when `dbapi.instrument_connection` signature changed but this is not updated here and few other places.